src/tls.c

```diff
@@ -4546,7 +4543,8 @@ static int TLSX_PointFormat_Parse(WOLFSSL* ssl, const byte* input,
 }
 
 #if defined(HAVE_ECC) || defined(HAVE_CURVE25519) || defined(HAVE_CURVE448)
-int TLSX_ValidateSupportedCurves(WOLFSSL* ssl, byte first, byte second) {
+int TLSX_ValidateSupportedCurves(const WOLFSSL* ssl, byte first, byte second,
```

Comment:
Why pass in CipherSuite when ecdhCurveOID is the only thing being set?

Reply:
Passing in only pointer to `ecdhCurveOID`.